### PR TITLE
Dependabot addition

### PR DESCRIPTION
**Pull request by arttarawork**

### CODE REVIEW

File: .vscode/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
```
Comment (TCourtneyOwen) [inline .vscode/dependabot.yml:8]: I am not familiar with dependabot but seems like daily may be too often. Can we do weekly?

Comment (arttarawork) [reply .vscode/dependabot.yml:8]: Changed to weekly with the latest check-in

**Comment by arttarawork** [comment]
Our pipeline tests are failing, but they're failing for reasons unrelated to this PR. The same test failure is occurring in the master branch of this repo.

File: .vscode/dependabot.yml (hunk 1 above)
Comment (akrantz) [reply .vscode/dependabot.yml:8]: Why is this in the .vscode folder? I don't think this is related to VS Code.

**Comment by akrantz** [comment]
If the tests on the master branch are not reliable, they need to be made reliable before merging more PRs.

File: .vscode/dependabot.yml (hunk 1 above)
Comment (TCourtneyOwen) [reply .vscode/dependabot.yml]: Yes, I agree with Adam, this seems strange

Comment (arttarawork) [reply .vscode/dependabot.yml]: Tests have moved to the proper .github folder, per Dependabot instructions: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/enabling-and-disabling-version-updates

**Comment by TCourtneyOwen** [comment]
All the CI agents are down I believe due to the Building 36 power outage over the weekend

**Comment by millerds** [comment]
This isn't needed.

**Pull request closed without merging**